index.html

```diff
@@ -0,0 +1,93 @@
+            </p>
+         </section>    
+    </main>
+    <section class="collabBoton"><p> start colaboration </p></section>
+    <div class="art"><img src="images/art.png" alt="artistic forms" width="320px"></div>
+    <section class="projects">
+        <h4>Projects</h4>
+        <img src="images/project1.png" alt="project1" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project2.png" alt="project2" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project3.png" alt="project3" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project4.png" alt="project4" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project 4.png" alt="project5" width="320px">
+        <div class="pData">
+            <h2> Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+    </section>
+    <h4>About</h4>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    <section class="collabBoton"><p> Get my CV &nbsp;&nbsp; <u>↓</u>   </p></section>
+    <footer>
+        <ul class="rsFooter" >
+            <li class="rrss"><img src="images/redes sociales/Vector (3).png" alt="GitHUB link"> </li>
+            <li class="rrss"><img src="images/redes sociales/Vector (4).png" alt="Twitter link"></li>
+            <li class="rrss"><img src="images/redes sociales/Vector (5).png" alt="LinkedIn link"></li>
+            <li class="rrss"><img src="images/redes sociales/Vector (6).png" alt="M link"></li>
+            <li class="rrss"><img src="images/redes sociales/Vector (7).png" alt="RabitHand link"></li>
+        </ul>
```

Comment:
- [ ] You still need to add Frameworks, languages and skills elements according to the requirements. With what I noticed, you've not added it to your about section. 👍 

## Figma Design

![crr31](https://user-images.githubusercontent.com/103386183/198850919-9f06ecc6-5430-4cb3-a257-ba409e5288b6.png)